### ISSUE TITLE
Commons help page is not mobile friendly

### ISSUE THREAD

**Issue by domdomegg** [commons-app/apps-android-commons#2145]
**Summary:** 

At the end of the tutorial there is a 'More information' link which takes the user to the help pages of the commons site. This landing page has a load of boxes with fixed sizes which are annoying to browse on mobile, not a great experience.

**Steps to reproduce:** 

1. Go to https://commons.m.wikimedia.org/wiki/Help:Contents on a mobile device

**Screen-shots:** 

![commons m wikimedia org_wiki_help_contents](https://user-images.githubusercontent.com/4953590/50082365-aa236e80-01e8-11e9-8086-d491cf56138d.png)

**Would you like to work on the issue?**

Yes

**Comment by nicolas-raoul** [comment]
How do you plan to solve this?
A first step could be to explain the problem on the talk page of that article. I am sure the developers thought this was already "mobile enough".
I see two urgent responsive design problems:
- Let the grey area be as narrow as the screen.
- Make the yellow and blue areas go on top of each other rather than always being side-by-side.

**Comment by domdomegg** [comment]
Fixed the grey area as long as the width > 335px.

No idea how to deal with boxes given I don't think we can use media queries. Boring way would probably be simply removing the table layout and have boxes under each other in all scenarios.

Also might be an idea to have the languages be a dropdown or at least have the title at the top rather than on the side:

![screenshot from 2018-12-17 11-08-59](https://user-images.githubusercontent.com/4953590/50083584-2e2b2580-01ec-11e9-83f7-c3629582ea96.png)

**Comment by domdomegg** [comment]
In fact on mobile we can hide the translation box completely given that there's already a translate button in the header.

**Comment by domdomegg** [comment]
I've added a comment on the [talk page](https://commons.wikimedia.org/wiki/Help_talk:Contents). If nothing happens for a week may want to ask for help from the Help Desk or Village Pump.

**Comment by sivaraam** [comment]
I believe this has been fixed now. I could see that the page is reasonably mobile friendly now. Can we close this?

**Comment by nicolas-raoul** [comment]
The two yellow and blue columns (= the main content) are still a pain to read :-/
In the age of mobile upstream should really find a solution to make the boxes responsive.

**Comment by sivaraam** [comment]
Yeah, I agree the page doesn't provide the best mobile friendly experience yet. I believe we could file a task about this in phabricator:

https://phabricator.wikimedia.org/maniphest/task/edit/form/10/?title=&projects=mobile
